frontend/src/components/annotation/annotation_ui_factory.vue

```diff
@@ -495,13 +497,21 @@ export default Vue.extend({
                 {name: "ellipse", display_name: "Ellipse & Circle", icon: "mdi-ellipse-outline"},
                 {name: "curve", display_name: "Curve Quadratic", icon: "mdi-chart-bell-curve-cumulative"},
             ],
+          current_ann_ctx: {
```

Comment:
Adding ` current_ann_ctx` reactive property to avoid using `annotation_ui_context.get_current_ann_ctx()` method